src/bindings/python/tests_compatibility/test_utils/test_utils.py

```diff
@@ -25,13 +25,17 @@ def generate_image(shape: Tuple = (1, 3, 32, 32), dtype: Union[str, np.dtype] =
     return np.random.rand(*shape).astype(dtype)
 
 
-def generate_model(input_shape: List[int]) -> openvino.inference_engine.ExecutableNetwork:
+def generate_relu_model(input_shape: List[int]) -> openvino.inference_engine.IENetwork:
     param = ng.parameter(input_shape, np.float32, name="parameter")
     relu = ng.relu(param, name="relu")
     func = Function([relu], [param], "test")
     func.get_ordered_ops()[2].friendly_name = "friendly"
-
-    core = IECore()
     caps = Function.to_capsule(func)
     cnnNetwork = IENetwork(caps)
+    return cnnNetwork
+
+
+def generate_relu_compiled_model(input_shape: List[int]) -> openvino.inference_engine.ExecutableNetwork:
+    core = IECore()
+    cnnNetwork = generate_relu_model(input_shape)
     return core.load_network(cnnNetwork, "CPU", {})
```

Comment:
should we replace CPU here with generic device name?

Reply:
done